pkg/ccl/sqlccl/csv_test.go

```diff
@@ -451,11 +490,21 @@ func TestImportStmt(t *testing.T) {
 				t.Fatalf("expected %d rows, got %d", expected, actual)
 			}
 
-			// Verify correct number of rows with RESTORE.
+			// Verify correct number of rows via COUNT.
 			sqlDB.QueryRow(`SELECT count(*) FROM t`).Scan(&result)
 			if expect := expectedRows; result != expect {
 				t.Fatalf("expected %d rows, got %d", expect, result)
 			}
+
+			// Verify correct number of NULLs via COUNT.
+			sqlDB.QueryRow(`SELECT count(*) FROM t WHERE b IS NULL`).Scan(&result)
+			expectedNulls := 0
+			if strings.Contains(tc.query, "nullif") {
```

Comment:
thanks, nice catch and it indeed looks like it too isn't quite plumbed... looking.